rest/model/stats.go

```diff
@@ -41,3 +41,47 @@ func (apiTestStats *APITestStats) BuildFromService(h interface{}) error {
 func (apiTestStats *APITestStats) ToService() (interface{}, error) {
 	return nil, errors.Errorf("ToService() is not implemented for APITestStats")
 }
+
+// APITaskStats is the model to be returned by the API when querying task execution statistics
+type APITaskStats struct {
+	TaskName     string `json:"task_name,omitempty"`
+	BuildVariant string `json:"variant,omitempty"`
+	Distro       string `json:"distro,omitempty"`
+	Date         string `json:"date"`
```

Comment:
use APIString and don't use omit empty.

Reply:
I updated the struct to use APIString and removed the omitempty on TaskName because it's not needed.
I kept the omitempty on the other field because they should not always be present in the results.